litert/tools/flags/vendors/qualcomm_flags.cc

```diff
@@ -384,6 +390,14 @@ Expected<void> UpdateQualcommOptionsFromFlags(QualcommOptions& opts) {
                 });
   opts.SetDumpTensorIds(int32_ids);
 
+  const auto skip_op_ids = absl::GetFlag(FLAGS_qualcomm_skip_op_ids);
+  std::vector<size_t> int_skip_op_ids;
+  std::for_each(skip_op_ids.begin(), skip_op_ids.end(),
+                [&int_skip_op_ids](const std::string& id) {
+                  int_skip_op_ids.push_back(std::stoi(id));
+                });
```

Comment:
Reserve `int_skip_op_ids` to `skip_op_ids.size()` before push_back.
The return type of `std::stoi` is `int`. However, this function is currently pushing the value to a `size_t` variable.

It may be worthwhile to revisit the datatype for skip op ids in the entire PR.
For this function, would it be appropriate to consider implementing `std::stoul`?

Also, would using `std::transform` along with `std::back_inserter` improve readability?